deploy/olm-catalog/kogito-operator/kogito-operator.package.yaml

```diff
@@ -1,7 +1,13 @@
 channels:
-- currentCSV: kogito-operator.v0.13.0
+- currentCSV: kogito-operator.v1.0.0-snapshot
   name: alpha
-- currentCSV: kogito-operator.v0.12.0
+- currentCSV: kogito-operator.v0.13.0
```

Comment:
moving dev-preview to 0.13.0 ?

Reply:
Already moved, there are lots of bug fixes in this version that's important to devpreview